

const addReviewController = async (req, res) => {

    try{

    } catch(error) {
        console.log(error);
        // return error status based on the error catched
    }

}

const deleteReviewController = async (req, res) => {

    try{

    } catch(error) {
        console.log(error);
        // return error status based on the error catched
    }

}

const updateReviewController = async (req, res) => {

    try{

    } catch(error) {
        console.log(error);
        // return error status based on the error catched
    }

}

const getReviewByIDController = async (req, res) => {

    try{

    } catch(error) {
        console.log(error);
        // return error status based on the error catched
    }

}

const getDoctorReviewsController = async (req, res) => {

    try{

    } catch(error) {
        console.log(error);
        // return error status based on the error catched
    }

}

module.exports = {
    addReviewController,
    updateReviewController,
    deleteReviewController,
    getDoctorReviewsController,
    getReviewByIDController
}